pyproject.toml

```diff
@@ -123,21 +119,65 @@ exclude_lines = [
     "if __name__ == .__main__.:",
 ]
 
-[tool.black]
-line-length = 100
-
-[tool.isort]
-profile = "black"
-src_paths = ["src", "tests"]
-
-
 [tool.pyright]
 include = ["src", "tests"]
 
+[tool.ruff]
+src = ["src", "test"]
+line-length = 100
 
-[tool.pylint.'MESSAGES CONTROL']
-disable = "R0913"
-
+[tool.ruff.lint]
```

Comment:
It might be helpful to comment want these are, similar to what we do [here](https://github.com/GenomicMedLab/software-templates/blob/18668529beece1589dc5a830438d3e6a6a05e79d/python/%7B%7Bcookiecutter.project_slug%7D%7D/pyproject.toml#L66)